.github/workflows/test.yml

```diff
@@ -9,6 +9,11 @@ on: # rebuild any PRs and main branch changes
 jobs:
   build: # make sure build/ci work properly
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
does it change something about tests suite strategy ?
When 1st test fails, CI is stopped ?

Reply:
With this flag each Node version is tested independently (one failure doesn't stop the other tests)